charts/cluster-api-provider-openstack/README.md

```diff
@@ -0,0 +1,44 @@
+# Installing Cluster API Provider OpenStack
+
+<details>
+<summary>Helm</summary>
+
+```shell
+helm repo add unikorn-cloud-capi https://unikorn-cloud.github.io/helm-cluster-api
+helm repo update
+helm install cluster-api-provider-openstack --version v0.2.0
+```
+</details>
+
+<details>
+<summary>ArgoCD</summary>
+
+```yaml
+---
+
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: cluster-api-provider-openstack
+  namespace: argocd
+spec:
+  project: default
+  source:
+    repoURL: https://unikorn-cloud.github.io/helm-cluster-api
+    chart: cluster-api-provider-openstack
+    targetRevision: v0.2.0
+  destination:
+    server: ${TARGET_CLUSTER}
```

Comment:
This is undefined anywhere, or documented for that matter, probably needs setting to `https://kubernetes.default.svc` for documentation purposes.

Missing a namespace, plus we should probably add the "create namespace" option to the application